src/test/java/uk/ac/ebi/embl/template/TemplateEntryProcessorTest.java

```diff
@@ -93,8 +96,9 @@ public void executeEntryProcessTestWithTaxId(String taxId, TemplateInfo template
 
         SourceFeature sourceFeature=templateProcessorResultSet.getEntry().getPrimarySourceFeature();
         assertTrue(templateProcessorResultSet.getValidationResult().isValid());
-        assertEquals(sourceFeature.getTaxId(),Long.valueOf(taxId));
```

Comment:
- We did we stop setting the taxId. It is possible that if it is not set and if we try to load flat files without writing them into a file first that we will fail to persist them in ena-sequence-persistence.
- Logically, the sourceFeature is constructed from the information in the template and there should not be any reason (I know of) that should prevent us from setting the taxon object fully in the sourceFeature. In fact, this is what I would expect to happen.